DuckDuckGo/Updates/UpdateController.swift

```diff
@@ -101,6 +101,9 @@ final class UpdateController: NSObject, UpdateControllerProtocol {
             if oldValue != areAutomaticUpdatesEnabled {
                 userDriver?.cancelAndDismissCurrentUpdate()
                 try? configureUpdater()
+#if !DEBUG
+                checkForUpdateIfNeeded()
```

Comment:
Triggers a "manual" update when the setting changes. This bypasses the phased rollout group handling.